bats/tests/helpers/load.bash

```diff
@@ -20,14 +20,19 @@ source "$PATH_BATS_ROOT/bats-support/load.bash"
 source "$PATH_BATS_ROOT/bats-assert/load.bash"
 source "$PATH_BATS_ROOT/bats-file/load.bash"
 
-# "defaults.bash" *must* be sourced before the rest of the files
-source "$PATH_BATS_HELPERS/defaults.bash"
-source "$PATH_BATS_HELPERS/utils.bash"
 source "$PATH_BATS_HELPERS/os.bash"
+source "$PATH_BATS_HELPERS/utils.bash"
+
+# os.bash and utils.bash must be loaded before defaults.bash
```

Comment:
I'm not a fan of _… must be loaded before X_ because it doesn't say _why_ that's the case (which means if we end up changing code it might no longer be required, we'd never realize that).